lib/stormpath/rails/config/social_login_verification.rb

```diff
@@ -0,0 +1,41 @@
+module Stormpath
+  module Rails
+    module Config
+      class SocialLoginVerification
+        attr_reader :app,
+                    :facebook_app_id,
+                    :facebook_app_secret,
+                    :github_app_id,
+                    :github_app_secret,
+                    :google_app_id,
+                    :linkedin_app_id
+
+        def initialize(app_href)
+          @app = Stormpath::Rails::Client.client.applications.get(app_href)
+          initialize_directories
+        end
+
+        private
+
+        def initialize_directories
```

Comment:
does account store mappings have a search functionality? Maybe you could search all directories with social login providers. Check Java.